docs/ccoctl.md

```diff
@@ -55,11 +55,11 @@ This will write out public/private key files named `serviceaccount-signer.privat
 To set up an OpenID Connect provider in the cloud, run
 
 ```bash
-$ ccoctl aws create-identity-provider --name=<name> --region=<aws-region> --public-key-file=/path/to/public/key/file
+$ ccoctl aws create-identity-provider --name=<name> --region=<aws-region> --public-key-file=/path/to/public/key/file --create-private-s3-bucket
 
 ```
 
-where `name` is the name used to tag and account any cloud resources that are created; `region` is the aws region in which cloud resources will be created; and `public-key-file` is the path to a public key file generated using `ccoctl aws create-key-pair` as [above](#creating-rsa-keys).
+where `name` is the name used to tag and account any cloud resources that are created; `region` is the aws region in which cloud resources will be created; and `public-key-file` is the path to a public key file generated using `ccoctl aws create-key-pair` as [above](#creating-rsa-keys). `create-private-s3-bucket` is an optional parameter which can be used to create private S3 bucket with public CloudFront OIDC endpoint (More details [here](./sts-private-bucket.md)).  
```

Comment:
I have some concerns related to the steps in the [markdown file](https://github.com/openshift/cloud-credential-operator/blob/master/docs/sts-private-bucket.md) linked from this line. I'll follow up in Slack with you and @mtulio :slightly_smiling_face: